server/controllers/IndividualServices.js

```diff
@@ -54,15 +59,27 @@ module.exports.putLiveAirInterfaceTransimitterIsOn = function putLiveAirInterfac
 };
 
 module.exports.putLiveControlConstructExternalLabel = function putLiveControlConstructExternalLabel (req, res, next, body, mountName, user, originator, xCorrelator, traceIndicator, customerJourney) {
+  let startTime = process.hrtime();
+  let responseCode = responseCodeEnum.code.OK;
+  let responseBodyToDocument = {};
   IndividualServices.putLiveControlConstructExternalLabel(body, mountName, user, originator, xCorrelator, traceIndicator, customerJourney)
-    .then(function (response) {
-      utils.writeJson(res, response);
-    })
-    .catch(function (response) {
-      utils.writeJson(res, response);
-    });
+  .then(async function (responseBody) {
+    let mountName = req.url.match(/(?<=construct=)(\w)+/g);
+    let requrl =req.url.replace(mountName,"{mountName}" );
```

Comment:
Also , move the logic of finding the openAPI path a scope up , so that the variable shall be resused in both the try and the catch statement.